site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-exportMemberList.vm

```diff
@@ -0,0 +1,31 @@
+<div class ="portletBody">
+ <h3>Participant List for: $validator.escapeHtml($!siteTitle)</h3>
+ <div>Total: $participantListSize</div>
+#if ($!allowUpdate || $allowUpdateSiteMembership || $!viewRoster)
+  <form name="participantForm" id="participantForm" action="#toolForm("SiteAction")" method="post">
+   <table class ="listHier lines nolines"  cellpadding="0" cellspacing="0" border="0" summary ="$tlang.getString("sitegen.siteinfolist.partlist.summary")">
+    <tr>
+     <th id="id">
+      <a href="#toolLinkParam("SiteAction" "doSort_roster" "criterion=display_id")" title ="$tlang.getString("sitegen.siteinfolist.sortid")"> 
+       $tlang.getString("sitegen.siteinfolist.id")
+       #if ($currentSortedBy.equals("display_id")) 
```

Comment:
Agreee with Earle. If display_id is the participant, then make it clear by using 'participant_display_id'.